package.json

```diff
@@ -24,5 +24,8 @@
   "files": [
     "template",
     "template.json"
-  ]
+  ],
+  "dependencies": {
+    "react-router-dom": "^6.2.1"
```

Comment:
Why is this defined in both packageJSON and templateJSON? `react` and other packages are not added like that.